api/src/plugins/createTask.ts

```diff
@@ -16,13 +16,17 @@ function buildNoteContent(
     note += ` - ${category}`;
   }
 
-  note += "\n";
+  note += "\n\n";
 
   if (description) {
     note += `## Description\n`;
+    note += "\n";
     note += `${description}\n`;
   }
 
+  note += "\n";
+  note += "----\n";
```

Comment:
Maybe it makes more sense to move the `---` inside the `if (description)`? Because if there is no description in the note, then there is nothing to separate.

Reply:
Everything else looks great and it now works well!

Reply:
Hey ! Just moved the separator.

You right, looks better without description now =)